lib/ddtrace/monkey.rb

```diff
@@ -16,8 +16,10 @@ module Monkey
     #   can be call twice but should just do nothing the second time.
     # - patched?, which returns true if the module has been succesfully
     #   patched (patching might have failed if requirements were not here)
-    @patchers = { elasticsearch: Datadog::Contrib::Elasticsearch::Patcher,
-                  redis: Datadog::Contrib::Redis::Patcher }
+    @patchers = {
+      elasticsearch: Datadog::Contrib::Elasticsearch::Patcher,
+      redis: Datadog::Contrib::Redis::Patcher
```

Comment:
The sinatra integration is not a hack, it's just another Sinatra extension. If you use the extension, then it's patched. I don't know if it makes sense to put it in that list.